README.md

```diff
@@ -39,26 +39,24 @@ The following projects use IOV-Core (add yourself via a PR):
 | Project                                                      | Tech stack / JavaScript environments |
 | ------------------------------------------------------------ | ------------------------------------ |
 | [TokenFinder](https://github.com/iov-one/token-finder)       | TypeScript / React / Browsers        |
-| [iov-faucet](https://github.com/iov-one/iov-faucet)          | TypeScript / NodeJS 10               |
+| [iov-faucet](https://github.com/iov-one/iov-faucet)          | TypeScript / NodeJS 10.13            |
```

Comment:
[10.15 right now](https://github.com/iov-one/iov-faucet/blob/master/Dockerfile) but [could be changed to 12 or something like that](https://github.com/iov-one/iov-faucet/issues/50). It would be good if we just track the major version here.